drivers/gpio/gpio_smartbond.c

```diff
@@ -318,12 +413,16 @@ static const struct gpio_driver_api gpio_smartbond_drv_api_funcs = {
 		.wkup_regs = (volatile struct gpio_smartbond_wkup_regs *)		\
 						DT_INST_REG_ADDR_BY_NAME(id, wkup),	\
 		.wkup_trig_select = id,							\
+		GPIO_PM_DEVICE_CFG(.ngpios, DT_PROP(DT_NODELABEL(gpio##id), ngpios))	\
```

Comment:
this is still wrong. You can't use nodelabels in instance-based macros. Instance `0` may be `gpio1`.